lib/hash.js

```diff
@@ -5,6 +5,7 @@ var KEY = Buffer.alloc(16)
 var OUT = Buffer.alloc(8)
 
 hash.TERMINATE = 4
+hash.LENGTH = 32
```

Comment:
Is this `32` related to the offsets used in `index.js`? I'm referring to the following lines:

```js
var key = self._version.slice(offset, offset + 32)
var seq = varint.decode(self._version, offset + 32)
offset += 32 + varint.decode.bytes
```

Reply:
+1 on @ralphtheninja's question